RecoMuon/MuonIdentification/plugins/ME0SegmentMatcher.cc

```diff
@@ -0,0 +1,285 @@
+/** \file ME0SegmentMatcher.cc
+ *
+ * \author David Nash
+ */
+
+#include <RecoMuon/MuonIdentification/plugins/ME0SegmentMatcher.h>
+
+#include <FWCore/PluginManager/interface/ModuleDef.h>
+#include <FWCore/Framework/interface/MakerMacros.h>
+
+#include <DataFormats/Common/interface/Handle.h>
+#include <FWCore/MessageLogger/interface/MessageLogger.h> 
+
+#include <DataFormats/MuonReco/interface/ME0Muon.h>
+
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+
+#include "TrackingTools/Records/interface/TrackingComponentsRecord.h"
+
+#include "DataFormats/GeometrySurface/interface/Plane.h"
+#include "TrackPropagation/SteppingHelixPropagator/interface/SteppingHelixPropagator.h"
+#include "TrackPropagation/SteppingHelixPropagator/interface/SteppingHelixStateInfo.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+
+#include "DataFormats/GeometrySurface/interface/LocalError.h"
+
+
+#include "DataFormats/HepMCCandidate/interface/GenParticle.h"
+#include "TLorentzVector.h"
+
+#include "DataFormats/TrajectoryState/interface/LocalTrajectoryParameters.h"
+#include "TrackingTools/AnalyticalJacobians/interface/JacobianCartesianToLocal.h"
+#include "TrackingTools/AnalyticalJacobians/interface/JacobianLocalToCartesian.h"
+
+
+ME0SegmentMatcher::ME0SegmentMatcher(const edm::ParameterSet& pas) : iev(0){
+  produces<std::vector<reco::ME0Muon> >();  
+  X_PULL_CUT   = pas.getParameter<double>("maxPullX");
```

Comment:
use lower case (bouncing cap, possibly with "the" or an underscore) for data members